### PR TITLE
Wrap lines

### LINKED ISSUE #83: Wrap all lines longer than 100 characters
Every line in code should be wrapped by the [100th character](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit) at the latest. There are some lines with the annotation `@link` in javadoc which contain more than 100 characters. See #82

### PR DESCRIPTION
**Pull request by bonaparten**
### Description
Wraps all line longer than 100 characters.

### Corresponding issue
Closes #83 

### New or changed dependencies
-

### Checklist
- [x] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#code-style) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/)
~- [ ] I have commented my code~
~- [ ] I have written javadoc (required for public methods)~
~- [ ] I have added sufficient unit and API tests~
~- [ ] I have made corresponding changes to the [documentation](https://github.com/GIScience/ohsome-api/tree/master/docs)~
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-api/blob/master/CHANGELOG.md)
~- [ ] I have adjusted the examples in the [check-ohsome-api](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api) script or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api/-/issues/new) in the corresponding repository. More Information [here](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#check-examples).~

### CODE REVIEW

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/Application.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public static ApplicationContext getApplicationContext() {
   /**
    * Main method to run this SpringBootApplication.
    * 
-   * @throws RuntimeException if database and keytables are not defined in the
+   * @throws RuntimeException if database and keytables are not defined in the
```
Comment (tyrasd) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/Application.java:49]: trailing whitespaces should be avoided

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,7 +21,9 @@ Changelog
 * restructure packages and classes within the controller and output packages ([#117])
 * extend docs on contribution types ([#134])
 * round coordinates of returned OSM features to 7 decimal places ([#138])
+* wrap all lines longer than 100 characters ([#83])
```
Comment (tyrasd) [inline CHANGELOG.md:24]: maybe to make it clear that it's about the code lines (and not the output json/csv lines for example):

```suggestion
* improve code style by wrapping all code lines longer than 100 characters ([#83])
```

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/elements/AreaController.java [hunk 3]
```diff
@@ -128,8 +128,8 @@ public Response areaGroupByBoundaryGroupByTag(HttpServletRequest servletRequest,
    * @param servletResponse <code>HttpServletResponse</code> of the outgoing response
    * @return {@link org.heigit.ohsome.ohsomeapi.output.Response Response}
    * @throws Exception thrown by {@link org.heigit.ohsome.ohsomeapi.executor.ElementsRequestExecutor
-   *                   #aggregateGroupByKey(RequestResource, HttpServletRequest,
-   *                   HttpServletResponse, boolean, boolean) aggregateGroupByKey}
+   *         #aggregateGroupByKey(RequestResource, HttpServletRequest, HttpServletResponse,
```
Comment (tyrasd) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/elements/AreaController.java:131]: trailing whitespace

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/elements/AreaController.java [hunk 4]
```diff
@@ -151,8 +151,8 @@ public Response areaGroupByKey(HttpServletRequest servletRequest,
    * @param servletResponse <code>HttpServletResponse</code> of the outgoing response
    * @return {@link org.heigit.ohsome.ohsomeapi.output.Response Response }
    * @throws Exception thrown by {@link org.heigit.ohsome.ohsomeapi.executor.ElementsRequestExecutor
-   *                   #aggregateGroupByTag(RequestResource, HttpServletRequest,
-   *                   HttpServletResponse, boolean, boolean) aggregateGroupByTag}
+   *         #aggregateGroupByTag(RequestResource, HttpServletRequest, HttpServletResponse,
```
Comment (tyrasd) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/elements/AreaController.java:154]: trailing whitespace

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/elements/AreaController.java [hunk 5]
```diff
@@ -201,8 +200,8 @@ public Response areaDensity(HttpServletRequest servletRequest,
    * @param servletResponse <code>HttpServletResponse</code> of the outgoing response
    * @return {@link org.heigit.ohsome.ohsomeapi.output.Response Response}
    * @throws Exception thrown by {@link org.heigit.ohsome.ohsomeapi.executor.ElementsRequestExecutor
-   *                   #aggregateGroupByType(RequestResource, HttpServletRequest,
-   *                   HttpServletResponse, boolean, boolean) aggregateGroupByType}
+   *         #aggregateGroupByType(RequestResource, HttpServletRequest, HttpServletResponse,
```
Comment (tyrasd) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/elements/AreaController.java:203]: trailing space

**Review by tyrasd — CHANGES_REQUESTED** [review]
I marked a few spots where this change introduces trailing whitespaces (i.e. a superfluous space ` ` at the end of a line), but didn't mark all of them because there were so many :see_no_evil: . Normally, IDEs remove those automatically during save, maybe yours doesn't?!

I can still see 6 lines over 100 characters. https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-148/4/checkstyle/type.-1710651122/#issuesContent
It seems like a good moment to fix these as well.

**Comment by bonaparten** [comment]
> I can still see 6 lines over 100 characters. https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-148/4/checkstyle/type.-1710651122/#issuesContent
> It seems like a good moment to fix these as well.

Two of them are in commented tests. I have to uncomment the tests and check if they work. I will do that in another PR.

**Review by tyrasd — APPROVED** [review]

**Pull request merged**